frontend/utils/fetcher.ts

```diff
@@ -23,8 +23,8 @@ export const textFetcher = async (input: RequestInfo, init: RequestInit) => {
   return res.text()
 }
 
-export const fetcher = async (input: RequestInfo, init: RequestInit) => {
-  const res = await fetch(input, init)
+export const fetcher = async (path: any) => {
```

Comment:
With how we've implemented our fetcher we should restrict `path` to being a `string`. Also consider using `url` instead of `path` as this is the common terminology used in the swr docs